README.md

```diff
@@ -36,11 +36,12 @@ This example partly mimics the steps of [the official LiveDNS documentation exam
 Note: sharing_id is optional. It is used e.g. when the API key is registered to a user, where the domain you want to manage is not registered with that user (but the user does have rights on that zone/organization).
 
 ```terraform
+
 terraform {
   required_providers {
     gandi = {
-      version = "~> 2.0.0"
-      source  = "github/go-gandi/gandi"
+      source = "go-gandi/gandi"
+      version = "2.0.0"
```

Comment:
I think we could still keep `~>` in order to support next minor release.

Reply:
Edited.